ports/stm32/eth.c

```diff
@@ -532,6 +605,10 @@ STATIC int eth_tx_buf_get(size_t len, uint8_t **buf) {
     *buf = &eth_dma.tx_buf[eth_dma.tx_descr_idx * TX_BUF_SIZE];
     tx_descr->tdes2 = len & TX_DESCR_2_B1L_Msk;
     tx_descr->tdes0 = (uint32_t)*buf;
+    #elif defined(STM32H5)
+    *buf = &eth_dma.tx_buf[eth_dma.tx_descr_idx * TX_BUF_SIZE];
```

Comment:
This looks the same as for H7, can it be combined?

Reply:
Done